ansible/docker_site.yml

```diff
@@ -12,9 +12,11 @@
     - {role: ci/init/create_workspace, when: run_install, tags: ["create_workspace"]}
     - {role: ci/update_dependencies, when: run_build, tags: ["update_dependencies", "build", "unit_tests", "code_coverage", "codecov_tool"]}
```

Comment:
need to add python_code_coverage and cpp_code_coverage tages to update_dependencies, build, unit_tests and shippable_finalize roles